docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  app:
+    image: friendsofsymfony/fos-http-cache
```

Comment:
having both a local config and a uploaded image seems like it could lead to confusion. if we use this on travis-ci we will quickly notice when the build no longer works, so i would vote for having a local docker file in this repo.